.github/.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-    ".": "5.0.0"
```

Comment:
please remove references from version changes and let the maintainers make these changes